### PR TITLE
Modfiy to create v1 docs

### PR DESCRIPTION
**Pull request by mj850**

### CODE REVIEW

File: pages/index.mdx [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 import { GanttChartSquare, Wallet, Wrench } from "lucide-react";
 
 # Introducing Sei
 
-Sei is the first parallelized EVM. This allows Sei to get the best of Solana and Ethereum - a hyper optimized execution layer that benefits from the tooling and mindshare around the EVM.
+<Callout>
+  Looking to build on our EVM Compatible Sei V2 Devnet?
+  Visit the Sei V2 Devnet docs [here](https://v2.docs.sei.io/).
+</Callout>
+Sei is the fastest parallel Layer 1 blockchain, optimized for trading.
```
Comment (besated) [inline pages/index.mdx:10]: think we might need to update the language here - let's consult growth on this

File: pages/getting-tokens/mainnet-tokens.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+### Wormhole 
+When bridging through Wormhole's Portal Bridge, your source assets are locked in a smart contract and a Wormhole-wrapped asset is minted on the target chain. These wrapped assets can then be swapped on an exchange for other assets on the target chain. Wormhole can be used to transfer tokens in from other blockchains to Sei or to transfer Sei tokens out to other blockchains. 
+### Axelar
+Axelar is a blockchain that connects blockchains, enabling secure interchain communication and token transfers. When bridging through Axelar, your source assets are deposited into a one-time address. Axelar Network will then confirm the deposit and transfer the funds to the destination address on the appropriate chain.
+### Token address list
+The following table lists the address associated with tokens bridged to Sei through Wormhole. This can be used as a reference to verify the legitimacy of tokens when you bridge.
+
+<Callout emoji="ℹ️">
+  These tokens addresses are also available in our registry on GitHub.
```
Comment (besated) [inline pages/getting-tokens/mainnet-tokens.mdx]: can you link to the registry here: https://github.com/sei-protocol/chain-registry/blob/main/assetlist.json

File: pages/interacting-with-sei/ecosystem-apps.mdx [hunk 3]
```diff
@@ -15,43 +14,37 @@ import seirumLogo from "../../public/assets/seirum.png";
   contact the project directly.
 </Callout>
 
-## Apps on `arctic-1`
+## Featured Dapps
```
Comment (besated) [inline pages/interacting-with-sei/ecosystem-apps.mdx]: this page might not be necessary i think we should just link to https://www.sei.io/ecosystem

File: pages/interacting-with-sei/signing-transactions.mdx [hunk 4]
```diff
@@ -1,63 +1,51 @@
...
 # Signing Transactions
 
 ## Introduction
 
-This guide provides instructions on how to sign both EVM and CosmWasm transactions on the Sei network using MetaMask. It also includes steps for viewing ERC20 tokens and ERC721 NFTs within the wallet.
+This guide provides instructions on how to sign CosmWasm transactions on the Sei network using Compass wallet.
 
-## Setting Up MetaMask for Sei
+## Setting Up your Sei wallet.
+You can use Compass or Leap wallet to sign trasactions on Sei.
```
Comment (besated) [inline pages/interacting-with-sei/signing-transactions.mdx]: lets do "Compass or Fin"

File: pages/interacting-with-sei/signing-transactions.mdx [hunk 5]
```diff
@@ -1,63 +1,51 @@
...
 
 # Signing Transactions
 
 ## Introduction
 
-This guide provides instructions on how to sign both EVM and CosmWasm transactions on the Sei network using MetaMask. It also includes steps for viewing ERC20 tokens and ERC721 NFTs within the wallet.
+This guide provides instructions on how to sign CosmWasm transactions on the Sei network using Compass wallet.
 
-## Setting Up MetaMask for Sei
+## Setting Up your Sei wallet.
```
Comment (besated) [inline pages/interacting-with-sei/signing-transactions.mdx]: nit: remove period at the end

File: pages/overview.mdx [hunk 6]
```diff
@@ -1,22 +1,33 @@
...
 
-Sei is the first parallelized EVM.
+Sei is a general purpose, open-source Layer 1 blockchain specialized for the exchange of digital assets. Leveraging a novel consensus and technical breakthroughs, Sei is the fastest blockchain in the industry. 
+<ImageWithCaption img={seiLogo} alt="Sei: The fastest Layer 1 blockchain" />
 
-<ImageWithCaption img={v2BannerImg} alt="Sei V2 Banner" />
+We are currently working on big changes in SeiV2, where we will introduce the first paralellized EVM (Ethereum Virtual Machine) compatible blockchain. This will allow Sei to get the best of Solana and Ethereum.
 
-# What is the EVM?
+Find out more on our [website](https://www.sei.io/)
```
Comment (besated) [inline pages/overview.mdx]: nit: add period at the end

File: pages/overview.mdx [hunk 7]
```diff
@@ -1,22 +1,33 @@
...
 
-The EVM (Ethereum Virtual Machine) is used to process transactions in blockchains such as Ethereum. Most crypto native developers are extremely familiar with the EVM.
+If you are an EVM developer, you can find the docs for our EVM compatible public devnet [here](https://v2.docs.sei.io/)
```
Comment (besated) [inline pages/overview.mdx]: nit: add period at the end

File: pages/overview.mdx [hunk 8]
```diff
@@ -1,22 +1,33 @@
 import { ImageWithCaption } from "../components";
 import v2BannerImg from "../public/assets/sei-v2-banner.jpg";
+import exchangeTrillema from "../public/assets/exchangeTrillema.png";
+import seiLogo from "../public/assets/sei-logo.png";
 
 # About Sei
 
-Sei is the first parallelized EVM.
+Sei is a general purpose, open-source Layer 1 blockchain specialized for the exchange of digital assets. Leveraging a novel consensus and technical breakthroughs, Sei is the fastest blockchain in the industry.
```
Comment (besated) [inline pages/overview.mdx]: same with language here, we'll likely have to consult with growth

File: theme.config.tsx [hunk 9]
```diff
@@ -20,6 +20,17 @@ const config: DocsThemeConfig = {
   feedback: {
     content: null,
   },
+  primaryHue: 0,
```
Comment (besated) [inline theme.config.tsx]: what are the default values of this? is this for the colors i assume

**Review by besated — CHANGES_REQUESTED** [review]

File: theme.config.tsx (hunk 9 above)
Comment (mj850) [reply theme.config.tsx]: image
The default is Blue - this primaryHue setting changes the colors of links and the nav bar on the side

Comment (mj850) [reply theme.config.tsx]: Actually blue seems a bit easier on the eyes. lmk what you think

File: pages/index.mdx (hunk 1 above)
Comment (mj850) [reply pages/index.mdx:10]: Updated overview page based on feedback

File: pages/block-explorers.mdx [hunk 10]
```diff
@@ -14,11 +14,15 @@ Block explorers offer tools for viewing and interacting with the network includi
...
   <Tabs.Tab>
 
-    - [Seistream](https://seistream.app/)
-    - [Seitrace](https://seitrace.com/)
+    - [Seiscan (Mainnet)](https://www.seiscan.app/pacific-1)
+
+  </Tabs.Tab>
+  <Tabs.Tab>
+
+    - [Seiscan (TestNet)](https://www.seiscan.app/atlantic-2)
```
Comment (besated) [inline pages/block-explorers.mdx]: typo: "Testnet" instead of "TestNet"

File: pages/getting-tokens/mainnet-tokens.mdx [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+| Osmosis                   | OSMO    | Osmosis                                     | ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518                                                |
+| Atom                      | ATOM    | Osmosis                                     | ibc/6CDD4663F2F09CD62285E2D45891FC149A3568E316CE3EBBE201A71A78A69388                                                |
+| Wrapped Ether (Wormhole)  | WETH    | Ethereum                                    | factory/sei189adguawugk3e55zn63z8r9ll29xrjwca636ra7v7gxuzn98sxyqwzt47l/4tLQqCLaoKKfNFuPjA9o39YbKUwhR1F8N29Tz3hEbfP2 |
+| USD Coin (Wormhole)       | USDCet  | Ethereum                                    | factory/sei189adguawugk3e55zn63z8r9ll29xrjwca636ra7v7gxuzn98sxyqwzt47l/Hq4tuDzhRBnxw3tFA5n6M52NVMVcC19XggbyDiJKCD6H |
+| Wrapped BTC (Wormhole)    | WBTC    | Ethereum                                    | factory/sei189adguawugk3e55zn63z8r9ll29xrjwca636ra7v7gxuzn98sxyqwzt47l/7omXa4gryZ5NiBmLep7JsTtTtANCVKXwT9vbN91aS1br |
+
```
Comment (besated) [inline pages/getting-tokens/mainnet-tokens.mdx:44]: i think we should also add native usdc (noble) and native usdt (kava) here to - https://github.com/sei-protocol/chain-registry/blob/71938fc346e65df7458e3458f16faab3faf198aa/assetlist.json#L478-L522

File: pages/interacting-with-sei/ecosystem-apps.mdx [hunk 12]
```diff
@@ -1,57 +1,11 @@
 import { Callout } from "nextra/components";
 import { Card, Cards } from "../../components";
-import camelLogo from "../../public/assets/camel.png";
-import gamblinoLogo from "../../public/assets/gamblino.jpeg";
-import dragonswapLogo from "../../public/assets/dragonswap.jpeg";
-import fluidLogo from "../../public/assets/fluid.png";
-import belugasLogo from "../../public/assets/belugas.png";
-import seirumLogo from "../../public/assets/seirum.png";
+import levanaLogo from "../../public/assets/logos/levana.jpeg";
```
Comment (besated) [inline pages/interacting-with-sei/ecosystem-apps.mdx]: i think u can remove these images now since we're just linking to ecosystem page

**Review by besated — APPROVED** [review]
left a couple comments, otherwise lgtm!

**Pull request merged**